regression-test/suites/load_p0/insert/test_insert.groovy

```diff
@@ -48,4 +48,24 @@ suite("test_insert") {
     """
 
     qt_sql1 "select * from ${insert_tbl} order by 1, 2, 3, 4"
+
+    def insert_tbl_dft = "test_insert_dft_tbl"
+    sql """ DROP TABLE IF EXISTS ${insert_tbl_dft}"""
+    sql """
+        CREATE TABLE ${insert_tbl_dft} (
+            `k1` char(8) DEFAULT "hello",
+            `k2` char(8) DEFAULT "world",
+            `k3` int(10) DEFAULT "0"
```

Comment:
add more column types to the test:
1. the column is date type, set its default value
2. the column is date type and its default value is a expr, current_timestamp

Reply:
@yiguolei If we use `current_timestamp` as default value in regression-test, will the result become unstable because the `current_timestamp` is changing all the time? Maybe we can use expr like `to_date` to produce a stable result.

Reply:
modified